<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PwnGenDlg</class>
 <widget class="QDialog" name="PwnGenDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/icons/app16.png</normaloff>:/icons/icons/app16.png</iconset>
  </property>
  <widget class="QLabel" name="lb1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password Length:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leng">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Password Pattern Options:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="low">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Lowercase letters (a,b,c,d,e.....z)</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="up">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Uppercase letters (A,B,C,D,E.....Z)</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="num">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Numbers (0,1,2,3,4...9)</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="punc">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Punctuations ( #, $, @, [, ), /, ; ,\,  _, - ... )</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Generated Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cpBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy to clipboard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="genBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cloBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cloBtn</sender>
   <signal>clicked()</signal>
   <receiver>PwnGenDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
